docs/source/create_channel/add_orderer.md

```diff
@@ -80,6 +80,57 @@ Use:
 This command will start a network of 4 orderers and 2 peers and 1 CLI, a container of the fifth orderer will be started
 as well, but is not a part of the network at this stage. This command also will create a channel named "mychannel" in which the 4 orderers and the 2 peers participate.
 
+## Use the `osnadmin` CLI to add the new orderer to the test channel
+
+The new orderer needs to run the following command:
+
+```shell
+export OSN_TLS_CA_ROOT_CERT=${PWD}/organizations/ordererOrganizations/example.com/tlsca/tlsca.example.com-cert.pem
+export ADMIN_TLS_SIGN_CERT=${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer5.example.com/tls/server.crt
+export ADMIN_TLS_PRIVATE_KEY=${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer5.example.com/tls/server.key
+
+osnadmin channel join --channelID [CHANNEL_NAME]  --config-block [CHANNEL_CONFIG_BLOCK] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
```

Comment:
but you need first to fetch the config block via `peer channel fetch`

Reply:
You are correct.
The fetching should be first.